app/neo/artifact/ProjectSelect.tsx

```diff
@@ -23,7 +23,7 @@ export const ProjectSelect = ({ onProjectChange, setDefaultValue, label, project
     setProject(project);
     onProjectChange(project);
   };
-  if (defaultValue !== project) {
```

Comment:
you cannot set anything else then defaultValue with this condition.
probably introduced with https://github.com/axonivy/neo/commit/bce421fb301b93b6fbae404039355f9389483011